app-ios/tutanota/Sources/Alarms/AlarmManager.swift

```diff
@@ -215,208 +121,57 @@ class AlarmManager {
         existingAlarms.remove(at: index)
       }
     default:
+      // There are no updates for alarms
```

Comment:
should this be fatal? can we make a report?

Reply:
I am not sure what is better: having invalid local state or crashing. If we change alarms API we need to phase out all old apps first anyway but they might crash first. What do you think we should do?

Reply:
I feel it's safer for us to make a report out of it, we'd have to move `reportError` from the `DesktopFacade` to `CommonNativeFacade` and implement it for ios and android. but I think it's a task for another day, there's probably a few use cases for this